packages/convex-helpers/index.ts

```diff
@@ -7,11 +7,13 @@
  */
 export async function asyncMap<FromType, ToType>(
   list: Iterable<FromType>,
-  asyncTransform: (item: FromType) => Promise<ToType>
+  asyncTransform: (item: FromType, index: number) => Promise<ToType>
 ): Promise<ToType[]> {
   const promises: Promise<ToType>[] = [];
+  let index = 0;
   for (const item of list) {
-    promises.push(asyncTransform(item));
+    promises.push(asyncTransform(item, index));
+    index += 1;
```

Comment:
other files got added here b/c I pushed the merged main to your branch, but this was the only addition